tests/test_cli/test_scaffold/test_skills.py

```diff
@@ -553,3 +555,60 @@ def teardown_class(cls):
             shutil.rmtree(cls.t)
         except (OSError, IOError):
             pass
+
+
+class TestScaffoldSkillHelpWithInvalidDirectory:
+    """Test --help works when validation fails."""
+
+    @classmethod
+    def setup_class(cls):
+        """Set the test up."""
+        cls.runner = CliRunner()
+        cls.agent_name = "myagent"
+        cls.resource_name = "myresource"
+        cls.cwd = os.getcwd()
+        cls.t = tempfile.mkdtemp()
+        dir_path = Path(PACKAGES)
+        tmp_dir = cls.t / dir_path
+        src_dir = cls.cwd / Path(ROOT_DIR, dir_path)
+        shutil.copytree(str(src_dir), str(tmp_dir))
+
+        os.chdir(cls.t)
+        result = cls.runner.invoke(
+            cli,
+            [*CLI_LOG_OPTION, "init", "--local", "--author", AUTHOR],
+        )
+        assert result.exit_code == 0
+        result = cls.runner.invoke(
+            cli,
+            [*CLI_LOG_OPTION, "create", "--local", cls.agent_name],
+            standalone_mode=False,
+        )
+        assert result.exit_code == 0
+
+        os.remove(Path(cls.t) / cls.agent_name / DEFAULT_AEA_CONFIG_FILE)
+
+    def test_exit_code_equal_to_1(self):
```

Comment:
Is this what you meant?

```suggestion
    def test_exit_code_equal_to_0(self):
```

Reply:
yeah right